scripts/write_gcr_to_parquet.py

```diff
@@ -0,0 +1,163 @@
+        **kwargs):
+    """Write out dataframe to Parquet file.
+
+    Parameters
+    ----------
+    df : Pandas DataFrame
+        Pandas DataFrame with the input catalog data to write out.
+    output_filename : str, optional
+        Prefix to be added to the output filename. Default is 'cat'.
+    parquet_scheme : str, optional   ['simple' or 'hive']
+            'simple' stores everything in one file per tract
+            'hive' stores one directory with a _metadata file and then
+                the columns partitioned into row groups.
+            Default is simple
+    parquet_engine : str, optional
+        Engine to write parquet on disk. Available: fastparquet, pyarrow.
+        Default is fastparquet.
+    parquet_compression : str, optional
+        Compression algorithm to use when writing Parquet files.
+        Potential: gzip, snappy, lzo, uncompressed. Default is gzip.
+        Availability depends on the engine used.
+    verbose : boolean, optional
+        If True, print out debug messages. Default is True.
+    """
+    if verbose:
+        print("Writing chunk {} to Parquet file.".format(df))
+    # Append iff the file already exists
+    parquet_append = append and os.path.exists(output_filename)
+    df.to_parquet(output_filename,
+                  append=parquet_append,
+                  file_scheme=parquet_scheme,
+                  engine=parquet_engine,
+                  compression=parquet_compression)
+
+
+if __name__ == "__main__":
+    from argparse import ArgumentParser, RawTextHelpFormatter
+    usage = """
+Produce Parquet output file from a GCR catalog.
+
+Example:
+
+To produce an output Parquet file from the 'dc2_object_run1.2p' GCR catalog:
+
+python %(prog)s dc2_object_run1.2p
+
+By default the output name will be 'dc2_object_run1.2p.parquet'.  You could specify a different one on the command line:
+
+python %(prog)s dc2_object_run1.2p --output_filename dc2_object.parquet
+
+You can also specify the Parquet scheme, engine, and compression to use.
+
+python %(prog)s
+    --parquet_scheme hive
+    --parquet_engine fastparquet
+    --parquet_compression gzip
+
+The selected engine needs to be installed on your machine to use.  E.g.,
+
+pip install fastparquet --user
+pip install pyarrow --user
+
+Potential compression algorithms are gzip (default), snappy, lzo, uncompressed.
+Availability depends on the installation of the engine used.
+"""
+    parser = ArgumentParser(description=usage,
+                            formatter_class=RawTextHelpFormatter)
+    parser.add_argument('reader', help='GCR catalog to read.')
+    parser.add_argument('--output_filename', default=None,
+                        help='Output filename')
+    parser.add_argument('--include_native', action='store_true', default=False,
+                        help='Include the native along with the non-native GCR catalog quantities')
+    parser.add_argument('--exclude_native', dest='include_native', action='store_false',
```

Comment:
Is this option for backward-compatibility or for other specific reasons? Since by default the native quantities are excluded, this option seems odd?